### ISSUE TITLE
when i initialize the database it exits the process

### ISSUE THREAD

**Issue by lomarcsdev** [tguichaoua/promised-sqlite3#24]
as a followup to my previous issue:
seems like it just exits when i run const db = await AsyncDatabase.open("./database.db");
ts code: 
```
import { AsyncDatabase } from "promised-sqlite3";

async function init() {
    
    try {
        console.log("hi")
        const db = await AsyncDatabase.open("./database.db");
        console.log(db)
        
        await db.run("CREATE TABLE Users (id TEXT PRIMARY KEY, blocked INTEGER)");
        await db.run("CREATE TABLE Guilds (id TEXT PRIMARY KEY, name TEXT NOT NULL, owner TEXT NOT NULL, premium INTEGER, settings BLOB)");
        await db.run("INSERT INTO Users (id) VALUES(?)", "1089638937217224846");

        await db.close();
    } catch (error) {
        console.error(error);
    }
}

init();
```
js code (build from ts):
```
var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
    return new (P || (P = Promise))(function (resolve, reject) {
        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
        step((generator = generator.apply(thisArg, _arguments || [])).next());
    });
};
import { AsyncDatabase } from "promised-sqlite3";
function init() {
    return __awaiter(this, void 0, void 0, function* () {
        try {
            console.log("hi");
            const db = yield AsyncDatabase.open("./database.db");
            console.log(db);
            yield db.run("CREATE TABLE Users (id TEXT PRIMARY KEY, blocked INTEGER)");
            yield db.run("CREATE TABLE Guilds (id TEXT PRIMARY KEY, name TEXT NOT NULL, owner TEXT NOT NULL, premium INTEGER, settings BLOB)");
            yield db.run("INSERT INTO Users (id) VALUES(?)", "1089638937217224846");
            yield db.close();
        }
        catch (error) {
            console.error(error);
        }
    });
}
init();

```
output: 
```
$ yarn initialize
yarn run v1.22.19
$ node build/initialize-database.js
hi
Done in 0.46s.
```

_Originally posted by @lomarcsdev in https://github.com/tguichaoua/promised-sqlite3/issues/21#issuecomment-1502246834_

**Comment by tguichaoua** [comment]
Fixed in `2.0.1`

**Comment by lomarcsdev** [comment]
ah thx. sorry for this. didnt know you didnt updated the package yet
